.gitignore

```diff
@@ -1 +1,2 @@
 build
+.vscode
```

Comment:
It is totally reasonable to drop the .vscode changes, but please split this out to a separate patch (consider it preapproved!)